docs/50_hub.md

```diff
@@ -0,0 +1,26 @@
+# Using the DANDI Hub
+
+[DANDI Hub](http://hub.dandiarchive.org) is a JupyterHub instance in the cloud to interact with the data stored in DANDI, and is free to use for exploratory analysis of data on DANDI.
+Note that DANDI Hub is not intended for significant computation, but provides a place to introspect Dandisets and to perform some analysis and visualization of data.
```

Comment:
I think here would be great to point to some tutorial/walkthrough or a video depicting general features of JupyterHub/Lab.  Later we could compliment with our own.  Do you know any good one?

Reply:
Here is one [YouTube video](https://www.youtube.com/watch?v=5pf0_bpNbkw&t=09m20s) I found that has a good chapter on JupyterLab starting at 9m20s.

Reply:
yeah -- seems to be nicely done, add a reference?

Reply:
I have added a sentence in the suggestion below.